Source/WebCore/platform/graphics/FontCascadeCache.cpp

```diff
@@ -118,8 +118,14 @@ Ref<FontCascadeFonts> FontCascadeCache::retrieveOrAddCachedFonts(const FontCasca
     newEntry = makeUnique<FontCascadeCacheEntry>(FontCascadeCacheEntry { WTFMove(key), FontCascadeFonts::create() });
     Ref<FontCascadeFonts> fonts = newEntry->fonts.get();
 
+
+#if !PLATFORM(IOS_FAMILY)
```

Comment:
I don’t think OS variant is the best way to select this behavior. Some iPhone models have more RAM than some Mac models. Meanwhile some non-iOS set top boxes have less.

Probably the best thing to do is to pick the maximum that would ever be useful and the minimum that would ever be acceptable, and scale between the two based on RAM size.